setup.py

```diff
@@ -0,0 +1,38 @@
+from setuptools import setup, find_packages
```

Comment:
Looks good. Since we are not going to deploy it on PyPI, there's no need for the `long_description` stuff.